docs/architecture/decisions/0002-convert-to-static.md

```diff
@@ -0,0 +1,104 @@
+   to a couple of S3 buckets and a CDN (or similar)
+ - Reduce hosting costs (k8s clusters, ELBs, databases)
+ - Reduce code maintenance cost (urgent python upgrades / security
+   vulnerabilities) and overall engineering time.
+
+#### What enables us to consider this option?
+
+ - Historically Careers used to have a Admin interface to submit University
+   Days. This has been replaced by a static file named `university_events.yml`
+ - Also Careers was setup for multi-locale support but only en-US was ever
+   implemented. There 're no plans to support more locales in the future.
+ - There's currently no site functionality which requires a dynamic back-end.
+   The site can get rebuilt and pushed as static HTML whenever
+   `university_events.yml` changes or a Job Posting in Greenhouse gets created,
+   updated or deleted.
+
+### Build and Hosting Options
+
+We discussed a number of options for generating the static site including:
+
+ - K8s cron jobs
+ - Apache Airflow / Google Cloud Composer
+ - GitHub Actions:
+ - TravisCI
+ - AWS Lambda
+ - CircleCI
+ - GitlabCI
+ - Jenkins
+   - Would work but we want to move away from our current Jenkins instance.
+
+The static site would be hosted in S3 with CloudFront CDN in front.
+
+Another suggested option was [Netlify](http://netlify.com/) which can take care
+of the generation and the hosting of the website. Since Netlify has been brought
+up a few times as a possible solution for other MEAO projects, it makes a good
+candidate to try with a lower profile site (compared to our other properties)
+like careers.m.o.
+
+Netlify offers a compelling set of features, including WebHooks (for directly
+triggering the build from Greenhouse #344), custom HTTP Headers for certain
+paths (like the RSS feeds #360).
+
+The issue motivating this decision, and any context that influences or
+constrains the decision.
+
+### Other key points taken into consideration
+
+ - The Green House integration includes some parse logic to normalize the format
+   of the posted jobs which we'll have to re-write / integrate to any static
+   site generation tool
+ - The Django site works just fine and it's well maintained. It will be upgraded
+   with minimal effort to Django 2.2 soon (see #339) which is an LTS release and
+   will be supported until at least April 2022
+ - We have very limited engineering time to devote to this effort
+
+
+## Decision
+
+We agreed to:
+ - Continue maintain the Django project and upgrade to Django 2.2 LTS release.
+ - Create scripts to generate a static copy of the Django site, along with the
+   required build acceptance tests to verify that the builds are correct.
+ - Use [Netlify](http://netlify.com/) to host a generated static copy of the
+   Django project.
+ - Setup a webhook from Greenhouse to Netlify to update the site whenever a Job
+   Post gets updated.
+
+## Consequences
+
+### Risks
+
+- Site does not update when Greenhouse updates:
```

Comment:
Troubleshooting should be documented as part of https://github.com/mozmeao/lumbergh/issues/365

Reply:
Agreed. I'm unclear whether you're suggesting changes in this pr or not?

Reply:
should be in a separate PR